tracer/src/Datadog.Trace/Configuration/ExporterSettings.cs

```diff
@@ -127,8 +149,8 @@ internal ExporterSettings(IConfigurationSource source, Func<string, bool> fileEx
 
         /// <summary>
         /// Gets or sets the unix domain socket path where the Tracer can connect to the Agent.
+        /// This parameter is deprecated and shall be removed. Consider using AgentUri instead
         /// </summary>
```

Comment:
```suggestion
        /// <summary>
        /// Gets or sets the unix domain socket path where the Tracer can connect to the Agent.
        /// </summary>
        [Obsolete("This API is deprecated. Use AgentUri instead with scheme 'unix://'")]
```

Reply:
Actually, I wanted to mark it Obsolete but Andrew rightfully raised that it was somehow a breaking change as some users could fail their builds on warnings.

Reply:
Yeah, treat errors as warnings is a best practice that some places enforce, so is technically breaking. I'm not passionate about it though 🙂